src/app/content/highlights/components/EditCard.tsx

```diff
@@ -71,6 +79,18 @@ const EditCard = React.forwardRef<HTMLElement, Props>((
     }
   };
 
+  const cancelEditing = React.useCallback(() => {
```

Comment:
don't put useCallback here, we don't need it to be memoized, that seems like its asking for trouble